pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -162,12 +162,20 @@ func prepareDHCPOptions(
 				if ip == nil {
 					return nil, fmt.Errorf(errorNTPConfiguration, server)
 				}
-
 				ntpServers = append(ntpServers, []byte(ip))
 			}
 
 			dhcpOptions[dhcp.OptionNetworkTimeProtocolServers] = bytes.Join(ntpServers, nil)
 		}
+
+		if customDHCPOptions.ExtraOptions != nil {
+			for _, extraOptions := range customDHCPOptions.ExtraOptions {
+				// parse string value of option to Integer value
```

Comment:
Do you mean parsing integer to byte?

Reply:
nah, that comment is irrelevant, I should have removed it, pardon me for the confusion. I added the comment when we were using map[string]string for the impl of the ExtraOptions.